First/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please add .class, .project, .settings and unnecessory files to .gitignore.

Reply:
Done ! Added above files to .gitIgnore file